src/Integration.Vsix.UnitTests/SonarAnalyzerCollisionManagerTests.cs

```diff
@@ -0,0 +1,163 @@
+//-----------------------------------------------------------------------
+// <copyright file="BoundSolutionAnalyzerTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.Vsix;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    [TestClass]
+    public class SonarAnalyzerCollisionManagerTests
+    {
+        #region Tests
+
+        [TestMethod]
+        public void SonarAnalyzerCollisionManager_HasNoCollidingAnalyzerReference_OnEmptyList()
+        {
+            var hasConflict = SonarAnalyzerCollisionManager.HasConflictingAnalyzerReference(null);
+            Assert.IsFalse(hasConflict);
```

Comment:
I would run Assert.IsFalse(SonarAnalyzerCollisionManager.HasConflictingAnalyzerReference(...)); Directly and provide a message i.e. Fails for null list/Fails for empty list.